ferry.go

```diff
@@ -426,7 +426,7 @@ func (f *Ferry) Initialize() (err error) {
 	if f.ErrorHandler == nil {
 		f.ErrorHandler = &PanicErrorHandler{
 			Ferry:             f,
-			DumpStateToStdout: true,
```

Comment:
The problem is that we have a conflict between sharding and copydb .

copydb defaults this variable to true while sharding defaults this variable to false. This is because we have two ways to dump statedumps, one via stdout and another via the http callback. 

We should reconcile this with each other. The only way to do this, likely, is to redesign how [this config var](https://github.com/Shopify/ghostferry/blob/master/config.go#L582-L585) is specified. We may need to add another variable, or maybe there's something else we can do to make this situation less confusing.